content/en/tracing/advanced/runtime_metrics.md

```diff
@@ -55,9 +55,44 @@ Coming Soon. Reach out to [the Datadog support team][1] to be part of the beta.
 {{% /tab %}}
 {{% tab "Ruby" %}}
 
-Coming Soon. Reach out to [the Datadog support team][1] to be part of the beta.
+<div class="alert alert-info">
+This feature is currently in <strong>BETA</strong>.
+Reach out to <a href="/help">the Datadog support team</a> to be part of the beta.
+</div>
+
+Runtime metrics collection uses the [`dogstatsd-ruby`][1] gem to send metrics to the Statsd agent. To collect runtime metrics, you must add this gem to your Ruby application.
```

Comment:
"to send metrics to the Statsd agent." -> "to send metrics via DogStatsD".  We dont offer a statsd agent, and in general we should refer to all agent processes as a single Agent.